scripts/98_MAF_quality_analysis.R

```diff
@@ -1,4 +1,4 @@
-# This scriptcontisns the investigation of MAF quality 
+# This scriptcontisns the investigation of MAF quality
```

Comment:
Here is a small typo. It should be "This script contains ... "

Reply:
fixed